stripe

```diff
@@ -0,0 +1 @@
+stripe
```

Comment:
you checked your symlink into git :) I didn't know this would happen either, but checking this in would cause problems for other people. just `git rm` it and push that commit.